src/app/shared/cookie-popup/cookie-popup.html

```diff
@@ -0,0 +1,13 @@
+<ng-template #content>
+  This site uses cookies from Google to deliver its services and to analyze traffic.
+
+  <div class="buttons">
+    <a
+      href="https://policies.google.com/technologies/cookies"
```

Comment:
How about providing policy link via input?
such as
```
<app-cookie-popup policy="https://policy.link" />
```

Reply:
This is a one-off component so it's fine to hardcode the link.